packages/serializer/package.json

```diff
@@ -14,10 +14,7 @@
   },
   "scripts": {
     "build": "ember build",
-    "lint:hbs": "ember-template-lint .",
```

Comment:
we can bring the lint command back if it turns out later we want it, but I suspect we need to set up a root config similar to eslint and prefer that.